src/facade/dragonfly_connection.cc

```diff
@@ -1017,7 +1020,7 @@ void Connection::DispatchFiber(util::FiberSocketBase* peer) {
       if (ShouldEndDispatchFiber(msg)) {
         RecycleMessage(std::move(msg));
         DCHECK(dispatch_q_.empty());
-        return;
+        break;
```

Comment:
This is not right, commented on this 😛 We set conn_closing below, we don't want it here